distribution/pom.xml

```diff
@@ -73,7 +73,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-assembly-plugin</artifactId>
-                        <version>3.6.0</version>
+                        <version>${maven-assembly-plugin.version}</version>
```

Comment:
I guess possibly even better is to add a pluginManagement section if we want to manage this centrally.

Reply:
I will do it with pleasure